applications/tari_console_wallet/src/ui/app.rs

```diff
@@ -145,24 +142,42 @@ impl<B: Backend> App<B> {
 
         self.tabs.draw_titles(f, title_halves[0]);
 
-        let chain_meta_data = match get_dummy_base_node_status() {
+        let base_node_state = self.app_state.get_base_node_state();
```

Comment:
This stuff should be moved into a proper self contained ui component that implements the `Component` trait and the drawing code should live in there I believe.

Reply:
Cool will do